bentoml/bentos.py

```diff
@@ -465,9 +465,15 @@ def containerize(
             )
         return False
     else:
-        logger.info(f'Successfully built docker image "{docker_image_tag}"')
+        custom_tags = False if not isinstance(docker_image_tag, str) else True
         logger.info(
-            f'To run your newly built Bento container, use: "docker run -it --rm -p 3000:3000 {docker_image_tag}"'
+            'Successfully built docker image for "%s" with tags "%s"',
+            str(bento.tag),
+            docker_image_tag if not custom_tags else ",".join(docker_image_tag),
```

Comment:
It seems to me like we should make this variable always an array.

Reply:
I remember we have this discussion before, let me do that.